src/appleseed/renderer/modeling/postprocessingstage/renderstamppostprocessingstage.cpp

```diff
@@ -142,31 +152,43 @@ namespace
                     static_cast<int>(props.m_canvas_height - 1)),
                 BackgroundColor);
 
-            // Draw the string into the image.
-            TextRenderer::draw_string(
-                frame.image(),
-                Font,
-                FontHeight,
-                FontColor,
-                MarginH + appleseed_seeds_16_width + MarginH,
-                origin_y,
-                text.c_str());
+            // Calculate final icon size and scale the icon
+            ImageSpec icon_spec = m_icon.spec();
+            float aspect = static_cast<float>(icon_spec.width) / icon_spec.height;
+            ROI roi(0, m_scale_factor * aspect * DefaultIconHeight, 0, m_scale_factor * DefaultIconHeight, 0, 1, 0, m_icon.nchannels());
+            ImageBuf scaled_logo;
+            ImageBufAlgo::resize(scaled_logo, m_icon, "mitchell", m_scale_factor, roi);
```

Comment:
Dark edges may be caused by the Mitchell-Netravali filter since it features negative lobes. Do dark edges disappear with other filters?